project/NoPublish.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, derived from Akka.
+ */
+
+package org.apache.pekko.grpc
+
+import sbt._
+import sbt.Keys._
+
+/**
+ * For projects that are not to be published.
+ */
+object NoPublish extends AutoPlugin {
```

Comment:
`SkipDocPublish`?

Reply:
I am just using the same name that is used in Pekko core. I also prefer `NoPublish` because its really a bit of a misnomer that modules which are not published still generate docs